docs/cloud-tutorials/cicd.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 7
+title: CI/CD tools
+---
+
+This this section we will cover tools provided by DBOS Cloud to enable CI/CD best practices.
+
+## Environments
+Use a different DBOS Cloud application for each environment, _e.g._, dev, staging and production.
+You can do so by [deploying](application-management#deploying-applications) the same application multiple times, each time with a different name, _e.g._, `my-app-dev`, `my-app-staging`, `my-app-prod`.
+
+:::info
+You must use a different database name (`app_db_name`) for each application.
```

Comment:
I think you want people to use a different database instance for dev and prod.

Reply:
Thanks!